static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -375,15 +381,16 @@ $gutter-large: 55px;
 .main-navigation {
     // Unset ul from _lists.scss
     margin-left: 0;
-    // Unset ul from user agent stylesheet
-    margin-top: 0;
+    // Unset ul from user agent stylesheet / Hides the line above the first 'pillar'
```

Comment:
fair enough! In that case just remove the `unset` part of this comment.

I was thinking we could just call them "primary Items"? But pillars works too